instrumentation/opentelemetry-instrumentation-aiohttp-client/src/opentelemetry/instrumentation/aiohttp_client/__init__.py

```diff
@@ -273,8 +274,6 @@ def instrumented_init(wrapped, instance, args, kwargs):
         if context_api.get_value(_SUPPRESS_INSTRUMENTATION_KEY):
             return wrapped(*args, **kwargs)
 
-        trace_configs = list(kwargs.get("trace_configs") or ())
```

Comment:
Does this mean we are removing the ability to pass the `trace_configs` from `aiohttp.ClientSession.__init__` and adding it to `.instrument()`?